lib/PostMessageTransport.js

```diff
@@ -83,6 +95,13 @@ define(function (require, exports, module) {
             PostMessageTransportRemote +
             "</script>\n";
     }
+
+    function reload() {
+        var launcher = Launcher.getCurrentInstance();
+        var liveDoc  = LiveDevMultiBrowser._getCurrentLiveDoc();
+
+        launcher.launch(liveDoc.doc.file.fullPath);
```

Comment:
As a reminder, `fullPath` is a private variable of a LiveDocument which they only expose for their unit tests.